kubernetes/src/test/java/oracle/kubernetes/operator/create/CreateOperatorGeneratedFilesTestBase.java

```diff
@@ -457,7 +457,17 @@ private V1ClusterRole getExpectedWeblogicOperatorClusterRole() {
             newPolicyRule()
                 .addApiGroupsItem("authorization.k8s.io")
                 .resources(singletonList("selfsubjectrulesreviews"))
-                .verbs(singletonList("create")));
+                .verbs(singletonList("create")))
```

Comment:
This creation of newPolicyRule is identical to the one at line 750, Please extract the two of them into a common method, and change the `asList` call to use `List.of()`